cmd/asset-syncer/utils.go

```diff
@@ -826,7 +833,11 @@ func (f *fileImporter) fetchAndImportIcon(c models.Chart, r *models.RepoInternal
 	// TODO: make this configurable?
 	resizedImg := imaging.Fit(img, 160, 160, imaging.Lanczos)
 	var buf bytes.Buffer
-	imaging.Encode(&buf, resizedImg, imaging.PNG)
+	err = imaging.Encode(&buf, resizedImg, imaging.PNG)
+	if err != nil {
+		log.WithFields(log.Fields{"name": c.Name}).WithError(err).Error("failed to encode icon")
+		return err
+	}
```

Comment:
Nice catch for the missing error.

Reply:
I can't take the credit here :P  the merit is on vscode and https://github.com/golangci/golangci-lint this time:

![image](https://user-images.githubusercontent.com/11535726/123988189-17bed100-d9c8-11eb-8cbd-7cac79e9f4e9.png)

Reply:
Mmm I'm dropping a card in the next interaction discussion for considering addressing all linter issues at once and adding it as part of the build process of our go dockerfiles